package.json

```diff
@@ -22,7 +22,8 @@
     "test": "make test"
   },
   "dependencies": {
-    "acorn": "^3.0.4",
+    "babylon": "^6.8.4",
+    "estree-walker": "0.2.1",
```

Comment:
Do we need to be pinning this to a specific version? I won't claim to be the authority, but I'd say it's probably more conventional for a utility to define its dependencies with some flexibility to allow for deduping of modules (consuming modules can use shrinkwrap if they want to pin specific versions)? Or at the very least, we should be consistent with how we're defining the dependencies.